src/views/template/factory/factory.html

```diff
@@ -2,6 +2,12 @@
     <div class="panel-heading">
         {{#l}}Factory{{/l}}
     </div>
+
+    <button type="button" onclick="GAME.model.factorycontroller.increaseTruckSize(order)">
```

Comment:
`[your button]` needs to be replaced with a proper jQuery selector. You may have to google! :)